### PR TITLE
Complete 4 tasks

### PR DESCRIPTION
**Pull request by ForceAwakenz**

### CODE REVIEW

File: task4.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    if (inputNumber % 1 > 0) {
+        return 'Real number, please input a natural number.';
+    }
+    if (inputNumber <= 0) {
+        return 'Negative number, please input a natural number';
+    }
+
+    if (inputNumber != 1) {
+        return inputNumber * factorial(inputNumber - 1);
+    } else { return 1; }
```
Comment (StanislawVictorovich) [inline task4.js:18]: please start return from the new line

File: task5.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    if (lastInSequence > 1) {
+        return lastInSequence + allSequenceTotal(lastInSequence - 1);
+    } else if (lastInSequence < -1) {
+        return lastInSequence + allSequenceTotal(lastInSequence + 1);
+    } if (lastInSequence == 1) {
+        return 1;
+    } else if (lastInSequence == -1) {
+        return -1;
+    } else { return 0;}
```
Comment (StanislawVictorovich) [inline task5.js:20]: the same here

File: task4.js (hunk 1 above)
Comment (ForceAwakenz) [reply task4.js]: done

File: task5.js (hunk 2 above)
Comment (ForceAwakenz) [reply task5.js]: done. found one more place and fixed there

**Review by StanislawVictorovich — APPROVED** [review]

**Pull request merged**